packages/langchain/src/orchestration/types.ts

```diff
@@ -20,3 +25,14 @@ export type OrchestrationCallOptions = Pick<
   tools?: Template['tools'];
   inputParams?: Prompt['inputParams'];
 };
+
+/**
+ * Orchestration module configuration with string templating.
+ * This type is used when the `templating` property can also be a string.
```

Comment:
Can also be or cannot be

Reply:
sorry I copied over the description from the old type.